masterlist.yaml

```diff
@@ -976,6 +1017,21 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5941/' ]
     after:
       - 'Relationship Dialogue Overhaul.esp'
+  - name: 'XPMSE.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/1988/'
+        name: 'XP32 Maximum Skeleton Special Extended - XPMSSE on Nexus Mods'
+    msg:
+      - <<: *doNotClean
+      - <<: *requireSKYUI
+        subs: [ '[XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988/)' ]
+        condition: 'file("scripts/XPMSEMCM.pex") and not active("SkyUI_SE.esp")'
+      - type: error
+        content: 'You have installed [XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988/) features which require [RaceMenu Special Edition](https://www.nexusmods.com/skyrimspecialedition/mods/19080/) to function correctly, but it is not installed.'
+        condition: 'file("scripts/RaceMenuPluginXPMSE.pex") and not active("RaceMenu.esp")'
+      - type: error
+        content: 'You have installed [XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988/) features which require [Fores New Idles in Skyrim SE](https://www.nexusmods.com/skyrimspecialedition/mods/3038) to function correctly, but it is not installed.'
+        condition: 'file("meshes/actors/character/character assets/skeleton.hkx") and not file("tools/GenerateFNIS_for_Users/GenerateFNISforUsers.exe")'
```

Comment:
For this scenario, there's one install option which doesn't add the animation files `skeleton.hkx` and ` skeleton_female.hkx.`. Those animations require FNIS `GenerateFNISforUsers.exe` to work correctly.
If another mod where to add those files, then it would flag. But in that case they're going to need the version of XP32 Maximum Skeleton with those animations anyway, so this shouldn't be an issue.